src/main/java/de/tum/in/www1/bamboo/server/StaticCodeAnalysisArtifacts.java

```diff
@@ -0,0 +1,24 @@
+package de.tum.in.www1.bamboo.server;
+
+/**
+ * The filename of the artifacts generated by Bamboo that are
+ * supported by the static code analysis parser.
+ */
+public enum StaticCodeAnalysisArtifacts {
+
+    SPOTBUGS("spotbugsXml.xml"),
```

Comment:
please add some documentation where these names are coming from and that they need to match what is stated in Bamboo in the artifact definition